res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -2266,74 +2244,32 @@ PioneerDDJSX.fxAssignButton = function(channel, control, value, status, group) {
     }
 };
 
-PioneerDDJSX.fxKnobMSB = function(channel, control, value, status) {
-    PioneerDDJSX.fxKnobMSBValue[channel - 4] = value;
-};
-
-PioneerDDJSX.fxKnobLSB = function(channel, control, value, status) {
-    var unit = channel - 4,
-        fullValue = (PioneerDDJSX.fxKnobMSBValue[unit] << 7) + value,
-        effect = ((control - 0x20) >> 1);
-
-    engine.setParameter(
-        "[EffectRack1_EffectUnit" + (unit + 1) + "_Effect" + effect + "]",
-        "meta",
-        fullValue / 0x3FFF
-    );
-};
-
-PioneerDDJSX.shiftFxKnobMSB = function(channel, control, value, status) {
-    PioneerDDJSX.shiftFxKnobMSBValue[channel - 4] = value;
-};
-
-PioneerDDJSX.shiftFxKnobLSB = function(channel, control, value, status) {
-    var unit = channel - 4,
-        fullValue = (PioneerDDJSX.shiftFxKnobMSBValue[unit] << 7) + value,
-        parameter = ((control - 0x30) >> 1) - 1,
-        fxEffectOn = false,
-        parameterLoaded = false,
-        group;
-
-    for (var effect = 0; effect < 3; effect++) {
-        group = "[EffectRack1_EffectUnit" + (unit + 1) + "_Effect" + (effect + 1) + "]";
-        fxEffectOn = engine.getValue(group, "enabled");
-
-        if (fxEffectOn) {
-            parameterLoaded = engine.getValue(group, "parameter" + (parameter + 1) + "_loaded");
-            if (parameterLoaded) {
-                engine.setParameter(group, "parameter" + (parameter + 1), fullValue / 0x3FFF);
-            }
-        }
-    }
-};
-
-PioneerDDJSX.fxBeatsKnob = function(channel, control, value, status, group) {
-    var unit = channel - 4,
-        delta = PioneerDDJSX.getRotaryDelta(value) / 30,
-        actValue,
-        newValue;
-
-    actValue = engine.getValue("[EffectRack1_EffectUnit" + (unit + 1) + "]", "super1");
-    newValue = actValue + delta;
-
-    if (newValue >= 0.00 && newValue <= 1.00) {
-        engine.setParameter("[EffectRack1_EffectUnit" + (unit + 1) + "]", "super1", newValue);
-    }
-};
-
-PioneerDDJSX.shiftFxBeatsKnob = function(channel, control, value, status, group) {
-    var unit = channel - 4,
-        delta = PioneerDDJSX.getRotaryDelta(value) / 30,
-        actValue,
-        newValue;
-
-    actValue = engine.getValue("[EffectRack1_EffectUnit" + (unit + 1) + "]", "mix");
-    newValue = actValue + delta;
+PioneerDDJSX.EffectUnit = function(unitNumber) {
```

Comment:
Whoops, [here is the example](https://mixxx.org/wiki/doku.php/components_js#setup1) I meant to link above.